sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -685,6 +685,17 @@ object SQLConf {
     .intConf
     .createWithDefault(100)
 
+  val MAX_PARAM_NUM_IN_JAVA_METHOD =
```

Comment:
I think this is not needed since we can use `CodeGenerator.calculateParamLength`

Reply:
ok